docs/rest_api.rst

```diff
@@ -312,6 +312,66 @@ Querying Information About Channels and Tokens
    :resjsonarr address partner_address: The partner we have a channel with
    :resjsonarr link channel: A link to the channel resource
 
+.. http:get:: /api/(version)/pending_transfers
+
+   Returns a list of all transfers that have neither been finished nor cancelled yet.
```

Comment:
So a transfer is finished when the task is removed from the list. Which makes your above statement about `finished` true. However, i would use the word "completed" rather than "finished".

Also, the cancelled transfer's task might still exist (Example, a transfer where the lock has expired but we keep the task around to send/receive the lock expiry message, or if the user cancel's a request when a refund was sent ... etc).
So, i would either remove the cancelled transfer from the above statement.